### ISSUE TITLE
ESM errors

### ISSUE THREAD

**Issue by Brian-McBride** [aesy/Easy-Bits#5]
Alas, Facebook continues to lag in supporting standards and there is an ESM error when trying to use Jest for testing in a mixed environment.

```
[PROJ_DIRECTORY]/node_modules/easy-bits/typings/easy-bits.d.ts:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,jest){export type Bit = 0 | 1 | false | true;

    SyntaxError: Unexpected token 'export'

[...]
    > 10 | import { BitField } from 'easy-bits';
```

This could be because there is no `dist` directory in the `node_modules` folder. This same thing happened with the release of `@faker-js/faker` today. Maybe a common change in some GitHub action not building before deploying to NPM?

**Comment by aesy** [comment]
Thanks for the report. This seems to be an error on my part, the dist folder was ignored on the last deploy (`1.2.1`). I'll push a fix later today, but in the meantime you can pin the version to `1.2.0`.

**Comment by Brian-McBride** [comment]
Thanks! 
And, I'll just add. Thanks for the lib. You really did make it super easy to work with bits :)

**Comment by aesy** [comment]
Aw, thank you! I'm happy you find it useful!

I've now published a new minor version, `1.2.2`, which includes the dist directory :facepalm:. Sorry about that!
